docs/source/markdown/podman-network-create.1.md

```diff
@@ -26,6 +26,14 @@ resolution.
 
 Driver to manage the network (default "bridge").  Currently only `bridge` is supported.
 
+#### **--opt**=*option*, **-o**
+
+Set driver specific options.
+
+For the `bridge` driver the following options are supported: `mtu` and `vlan`.
+The `mtu` option sets the Maximum Transmission Unit (MTU) and takes an integer value.
+The `vlan` option assign VLAN tag and enables vlan\_filtering. Defaults to none.
```

Comment:
Just triple checking, I think the '\` is intentional and needed?

Reply:
Sorry which one ? The backslash ? Editor was complaining about the "naked" slash, so I escaped it... Maybe needlessly so

Reply:
oops, messed up the md prior.  `vlan\_filtering` is what I was asking about.  Seems to resolve OK, I'm just not sure it's necessary.